doc/nrf/software_maturity.rst

```diff
@@ -323,7 +377,23 @@ The following table indicates the software maturity levels of the support for ea
 
 .. toggle::
 
-  .. sml-table:: zigbee
+   +-------------------------------------------+----------+----------+----------+----------+--------------+--------------+--------------+---------+---------+
+   |                                           | nRF52810 | nRF52811 | nRF52820 | nRF52832 |   nRF52833   |   nRF52840   |   nRF5340    | nRF9160 | nRF9161 |
+   +===========================================+==========+==========+==========+==========+==============+==============+==============+=========+=========+
+   | **OTA DFU over Zigbee**                   |    \-    |    \-    |    \-    |    \-    |      \-      | Experimental | Experimental |   \-    |   \-    |
+   +-------------------------------------------+----------+----------+----------+----------+--------------+--------------+--------------+---------+---------+
+   | **Zigbee (Sleepy) End Device**            |    \-    |    \-    |    \-    |    \-    | Experimental | Experimental | Experimental |   \-    |   \-    |
+   +-------------------------------------------+----------+----------+----------+----------+--------------+--------------+--------------+---------+---------+
+   | **Zigbee + Bluetooth LE multiprotocol**   |    \-    |    \-    |    \-    |    \-    | Experimental | Experimental | Experimental |   \-    |   \-    |
+   +-------------------------------------------+----------+----------+----------+----------+--------------+--------------+--------------+---------+---------+
+   | **Zigbee + nRF21540 (GPIO)**              |    \-    |    \-    |    \-    |    \-    |      \-      | Experimental |      \-      |   \-    |   \-    |
+   +-------------------------------------------+----------+----------+----------+----------+--------------+--------------+--------------+---------+---------+
+   | **Zigbee Coordinator**                    |    \-    |    \-    |    \-    |    \-    | Experimental | Experimental | Experimental |   \-    |   \-    |
+   +-------------------------------------------+----------+----------+----------+----------+--------------+--------------+--------------+---------+---------+
+   | **Zigbee Network Co-Processor (NCP)**     |    \-    |    \-    |    \-    |    \-    | Experimental | Experimental | Experimental |   \-    |   \-    |
+   +-------------------------------------------+----------+----------+----------+----------+--------------+--------------+--------------+---------+---------+
+   | **Zigbee Router**                         |    \-    |    \-    |    \-    |    \-    | Experimental | Experimental | Experimental |   \-    |   \-    |
+   +-------------------------------------------+----------+----------+----------+----------+--------------+--------------+--------------+---------+---------+
 
 Wi-Fi feature support
```

Comment:
Should we just consider removing this table on the 2.4 branch, since literally nothing is supported?

Reply:
[2.4.3](https://docs.nordicsemi.com/bundle/ncs-2.4.3/page/nrf/software_maturity.html#zigbee_feature_support) had it the same way.